dev-perl/Text-Soundex/Text-Soundex-3.05.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit perl-module
+
+DESCRIPTION="Implementation of the soundex algorithm"
+HOMEPAGE="http://search.cpan.org/~rjbs/Text-Soundex-3.05/Soundex.pm"
+SRC_URI="mirror://cpan/authors/id/R/RJ/RJBS/${P}.tar.gz"
```

Comment:
Instead of defining it this way, shouldn't you use ```DIST_AUTHOR``` and such from perl-module.eclass?